assets/dev/scss/frontend/_lightbox.scss

```diff
@@ -127,6 +125,25 @@
 		}
 	}
 
+	.swiper,
+	.swiper-container {
+
+		.elementor-swiper-button {
+
+			&-prev {
+				left: 0;
+			}
+	
+			&-next {
+				right: 0;
+			}
```

Comment:
I know it's not part of this job, you are just fixing swiper, but I wonder if these take into account RTL.
Maybe it's outside of the scope of this. Just thinking.

Reply:
@StuartElementor I think it does not matter because we are targeting BOTH left and right. If we just wanted to target left OR right then we needed to use some variables here :-) Do you agree?

Reply:
@StuartElementor I checked and the arrows are positioned in the exact same place in LTR and RTL at least, so I think this is okay..

Reply:
yah, I agree. Lets not change existing behaviour.